packages/sync-repo-settings/src/required-checks.json

```diff
@@ -109,10 +109,6 @@
       "googleapis/google-cloud-node"
     ],
     "repoOverrides": [
-      {
-        "repo": "GoogleCloudPlatform/nodejs-docs-samples",
```

Comment:
why did we delete this entry, wondering if it should be added back?

Reply:
Hey! I removed it because the rule is just to add in the branch protection that's already there. So, it would be the same as not updating it, I assume? Since we're not actually ever getting the native branch protection of any of the repos. Since the repo is also in the 'ignored repos' section, we really don't need to do anything to it.

Reply:
oh, as long as it's in ignored repos sounds good to me 👌